src/plots/cartesian/axes.js

```diff
@@ -63,6 +63,7 @@ axes.list = axisIds.list;
 axes.listIds = axisIds.listIds;
 axes.getFromId = axisIds.getFromId;
 axes.getFromTrace = axisIds.getFromTrace;
+axes.isLinked = axisIds.isLinked;
```

Comment:
Do we need to reexport this here? I think these `Axes` functions are relics of tight coupling to chart studio so should generally not be expanded.